example/src/guardian.prize.25220.ts

```diff
@@ -0,0 +1,471 @@
+import type { GuardianCrossword } from 'mycrossword';
+
+const data: GuardianCrossword = {
+    name: 'Prize crossword No 25,220',
+    crosswordType: 'prize',
+    id: 'prize/25220',
+    number: 25220,
+    dimensions: {
+      cols: 15,
+      rows: 15,
+    },
+    entries: [
+      {
+        id: '7-across',
+        number: 7,
+        humanNumber: '7',
+        position: {
+          x: 0,
+          y: 1,
+        },
+        length: 8,
+        clue: 'See 23',
+        solution: 'WITHOUTA',
+        direction: 'across',
+        group: ['7-across'],
+        separatorLocations: {},
+      },
+      {
+        id: '9-across',
+        number: 9,
+        humanNumber: '9',
+        position: {
+          x: 9,
+          y: 1,
+        },
+        length: 6,
+        clue: "The general view is it's a vice (6)",
+        solution: 'AGREED',
+        direction: 'across',
+        group: ['9-across'],
+        separatorLocations: {},
+      },
+      {
+        id: '10-across',
+        number: 10,
+        humanNumber: '10',
+        position: {
+          x: 0,
+          y: 3,
+        },
+        length: 4,
+        clue: 'See 23',
+        solution: 'NAME',
+        direction: 'across',
+        group: ['10-across'],
+        separatorLocations: {},
+      },
+      {
+        id: '11-across',
+        number: 11,
+        humanNumber: '11',
+        position: {
+          x: 5,
+          y: 3,
+        },
+        length: 10,
+        clue: 'See 23',
+        solution: 'WORTHANAGE',
+        direction: 'across',
+        group: ['11-across'],
+        separatorLocations: {},
+      },
+      {
+        id: '12-across',
+        number: 12,
+        humanNumber: '12',
+        position: {
+          x: 0,
+          y: 5,
+        },
+        length: 6,
+        clue: 'Weird agent holds love authorised (6)',
+        solution: 'SPOOKY',
+        direction: 'across',
+        group: ['12-across'],
+        separatorLocations: {},
+      },
+      {
+        id: '14-across',
+        number: 14,
+        humanNumber: '14',
+        position: {
+          x: 7,
+          y: 5,
+        },
+        length: 8,
+        clue: 'See 22',
+        solution: 'TOOSHORT',
+        direction: 'across',
+        group: ['14-across'],
+        separatorLocations: {},
```

Comment:
thanks! looking some more at this crossword it looks like our system has struggled with the separators on this crossword, I've manually set them for this example to something that at least looks plausibly correct

Reply:
@andrew-nowak out of interest, how does your system do this? Do you have some code that parses `.ccw` Crossword Compiler files? I only ask because this is something on my wish list and I haven't been able to find any libraries that do it.

Reply:
No, we don't use the `.ccw` files, we've always imported the `.xml` export format from Crossword Compiler. I wanted to double check if we can open source our service which handles crosswords (one of its responsibilities being the conversion from the XML to our JSON format), but it's slipped down the list of priorities at a busy time for us - I'll have another check when I get a chance

Reply:
Ah okay - same here. I've written quite an extensive set of import/export XML utils, probably like yourselves. Might be good to cross-reference the two... well, one day maybe.